### PR TITLE
Add docstring for roots

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

File: src/generic/Misc/Poly.jl [hunk 1]
```diff
@@ -140,6 +140,11 @@ end
 #
 ################################################################################
 
+@doc doc"""
+    roots(f::PolyElem)
+
+Returns the roots of the polynomial `f` in the base ring of `f` as an array.
```
Comment (rfourquet) [inline src/generic/Misc/Poly.jl:146]:
```suggestion
Return the roots of the polynomial `f` in the base ring of `f` as an array.
```

File: src/generic/Misc/Poly.jl [hunk 2]
```diff
@@ -151,6 +156,11 @@ function roots(f::PolyElem)
     return rts
 end
 
+@doc doc"""
+    roots(f::PolyElem, R::Field)
+
+Returns the roots of the polynomial `f` in the field `R` as an array.
```
Comment (rfourquet) [inline src/generic/Misc/Poly.jl:162]:
```suggestion
Return the roots of the polynomial `f` in the field `R` as an array.
```

**Comment by thofma** [comment]
Thank @rfourquet

**Comment by thofma** [comment]
Add it to the docs.

**Review by wbhart — APPROVED** [review]

**Pull request merged**